tests/utils/test_ensemble.py

```diff
@@ -34,11 +34,13 @@
     HinSAGELinkGenerator,
 )
 from stellargraph.data.converter import *
+from stellargraph.data import UnsupervisedSampler
 from stellargraph.utils import Ensemble, BaggingEnsemble
 
 from tensorflow.keras import layers, Model
 from tensorflow.keras.optimizers import Adam
 from tensorflow.keras.losses import categorical_crossentropy, binary_crossentropy
+from tensorflow import keras
```

Comment:
```suggestion
```
`keras.Model` (used later) is already imported.

Reply:
Yeah tru